en/core-api/credit-card.md

```diff
@@ -140,48 +142,36 @@ Sample onFailure `response` object, it may contains the `validation_messages`:
+#### Endpoints
 
-Requirement | Description
---- | ---
-Server Key | Explained on [previous section](/en/midtrans-account/overview.md)
-`order_id` | Transaction order ID, defined from your side
-`gross_amount` | Total amount of transaction, defined from your side
-`token_id` | Represents customer's card information acquired from [Get Card Token Response](en/core-api/credit-card.md#get-card-token-response)
-`authentication` | Flag to enable the 3D secure authentication.
-
-?> **Note**: For better security & fraud prevention, you should set `authentication` to `true`. Only set `false` if you have confirmed with Midtrans & acquiring bank
-
-### Charge API request
-
-The example below shows a sample codes of the charge request:
-<!-- tabs:start -->
-#### **API-Request**
+| Environment | Method | URL                                        |
+| ----------- | ------ | ------------------------------------------ |
+| Sandbox     | POST   | https://api.sandbox.midtrans.com/v2/charge |
+| Production  | POST   | https://api.midtrans.com/v2/charge         |
 
-*This is an example in Curl, please implement according to your backend language, you can switch to other language on the "tab" above. (you can also check our [available language libraries](/en/technical-reference/library-plugin.md))*
+#### Headers
 
-#### Request Details
-Type | Value
---- | ---
-HTTP Method | `POST`
-API endpoint (Sandbox) | `https://api.sandbox.midtrans.com/v2/charge`
-API endpoint (Production) | `https://api.midtrans.com/v2/charge`
+| Header Name   | Description                                            | Required | Values                |
+| ------------- | ------------------------------------------------------ | -------- | --------------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json      |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json      |
+| Authorization | The authentication method used to access the resource. | Required | Basic **AUTH_STRING** |
 
-#### HTTP Headers
-```
-Accept: application/json
-Content-Type: application/json
-Authorization: Basic AUTH_STRING
-```
+**AUTH_STRING**: Base64(`ServerKey + :`)<br>Midtrans API validates HTTP request using Basic Authentication method. The username is your *Server Key* while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username and password separated by a colon symbol (**:**). For more details, refer to [ API Authorization and Headers](https://docs.midtrans.com/en/technical-reference/api-header).
 
-**AUTH_STRING**: Base64(`ServerKey + :`)
+?> ***Note***: *Server Key* is required to authenticate the request. For more details, refer to [HTTPS Header](https://api-docs.midtrans.com/#http-s-header).
```

Comment:
These manual configuration of Endpoints, Headers etc details only relevant to CURL tabs, it is not relevant for other language, so it should not be displayed outside CURL tabs. I've made similar feedback on Madhura's work

Reply:
Resolved